lib/js/components/SurveyQuestions/SurveyQuestionScale/SurveyQuestionScale.vue

```diff
@@ -205,7 +205,7 @@ export default {
 		},
 		elaborationLabel: {
 			type: String,
-			required: true,
+			default: null,
```

Comment:
should we update storybook to reflect default state of component?

Reply:
I don't think so, it's already far from the default.